node/declaration.ts

```diff
@@ -30,19 +30,44 @@ export const getDeclarationsFromConfig = (
   // a function configuration object, we replace the path because that object
   // takes precedence.
   for (const declaration of [...tomlDeclarations, ...deployConfig.declarations]) {
-    const config = functionsConfig[declaration.function] ?? {}
+    const config = functionsConfig[declaration.function]
+
+    // If no config is found, add the declaration as is
+    if (!config) {
+      declarations.push(declaration)
+
+      // If we have path specified create a declaration for each path
+    } else if (config.path?.length) {
+      // eslint-disable-next-line max-depth
+      if (!Array.isArray(config.path)) config.path = [config.path]
+
+      config.path.forEach((path) => {
+        declarations.push({ ...declaration, ...config, path })
+      })
+
+      // If only cache was specified, add it to the declaration
```

Comment:
I don't think this comment is accurate anymore, since we're not targeting the `cache` property specifically?

Reply:
✅